docs/reference/connector/apis/update-connector-scheduling-api.asciidoc

```diff
@@ -0,0 +1,113 @@
+[[update-connector-scheduling-api]]
+=== Update connector scheduling API
+
+preview::[]
+
+++++
+<titleabbrev>Update connector scheduling</titleabbrev>
+++++
+
+Updates the `scheduling` configuration of a connector.
+
+[[update-connector-scheduling-api-request]]
+==== {api-request-title}
+
+`PUT _connector/<connector_id>/_scheduling`
+
+[[update-connector-scheduling-api-prereq]]
+==== {api-prereq-title}
+
+* To sync data using connectors, it's essential to have the Elastic connectors service running.
+* The `connector_id` parameter should reference an existing connector.
+
+[[update-connector-scheduling-api-path-params]]
+==== {api-path-parms-title}
+
+`<connector_id>`::
+(Required, string)
+
+[role="child_attributes"]
+[[update-connector-scheduling-api-request-body]]
+==== {api-request-body-title}
+
+`scheduling`::
+(Required, object) The scheduling configuration for the connector. This configuration determines frequency of synchronization operations for the connector.
+
+The scheduling configuration includes the following attributes, each represented as a `ScheduleConfig` object:
+
+- `access_control` (Required, `ScheduleConfig` object) Defines the schedule for synchronizing access control settings of the connector.
+
+- `full` (Required, `ScheduleConfig` object) Defines the schedule for a full content syncs.
+
+- `incremental` (Required, `ScheduleConfig` object) Defines the schedule for incremental content syncs.
+
+Each `ScheduleConfig` object includes the following sub-attributes:
+
+  - `enabled` (Required, boolean) A flag that enables or disables the scheduling.
+
+  - `interval` (Required, string) A CRON expression representing the sync schedule. This expression defines the grequency at which the sync operations should occur. It must be provided in a valid CRON format.
```

Comment:
> a valid CRON format

Is there a specific CRON flavor we can point users to?

Reply:
Yeah, we use the validation using a Class that is already defined in ES: https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/scheduler/Cron.java 

This implementation is compatible with the CRON expressions that we used so far for connectors. I'm not sure if it follows a particular flavor. I thing calling it `CRON` is a good starting point, as the validation errors coming from this class are very helpful explaining what's wrong.